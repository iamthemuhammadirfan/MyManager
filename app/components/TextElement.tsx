import {StyleSheet, Text, View} from "react-native";
import React from "react";

type Props = {
  children: string;
};
const TextElement = ({children}: Props) => {
  return <Text>{children}</Text>;
};

export default TextElement;

const styles = StyleSheet.create({});
