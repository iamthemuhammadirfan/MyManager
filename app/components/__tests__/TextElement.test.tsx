import React from "react";
import {render} from "@testing-library/react-native";

import TextElement from "components/TextElement";

describe("Text Element", () => {
  it("should be matched with snapshots", () => {
    const tree = render(<TextElement>Hi</TextElement>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
