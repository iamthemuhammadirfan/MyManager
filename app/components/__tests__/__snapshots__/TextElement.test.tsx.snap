// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Element should be matched with snapshots 1`] = `
<Text>
  Hi
</Text>
`;
